src/pages/FishListPage.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+import FishCard from '../components/FishCard.js';
+import Header from '../components/Header.js';
+
+export default function FishList({ fishes }) {
```

Comment:
I would name the function the same as the file, so in this case it would be FishListPage.

As you only have one page, you also could have written this code into the App directly. But as you probably need it later on anyway I guess it's okay like that.